pop/service/service_no_race_test.go

```diff
@@ -0,0 +1,81 @@
+// +build !race
```

Comment:
I think it should build without races - look at it as an exercise in concurrent programming - very important when building distributed systems ;)